router.go

```diff
@@ -122,6 +122,7 @@ func NewRouter(server *Server) *Router {
 	for i := 400; i <= 418; i++ {
 		router.StatusHandler(&ErrorStatusHandler{}, i)
 	}
+	router.StatusHandler(&RequestErrorStatusHandler{}, http.StatusBadRequest)
```

Comment:
```suggestion
	for i := 401; i <= 418; i++ {
		router.StatusHandler(&ErrorStatusHandler{}, i)
	}
	router.StatusHandler(&RequestErrorStatusHandler{}, http.StatusBadRequest)
```

Reply:
What about 

from := http.StatusUnauthorized // 401
to := http.StatusTooManyRequests // 429

for i := from; i < to; i++

Reply:
What about 

from := http.StatusUnauthorized // 401
to := http.StatusTooManyRequests // 429

for i := from; i < to; i++

Reply:
Renamed all.